src/lib/drupal/lovell/utils.ts

```diff
@@ -133,3 +133,34 @@ export function isLovellBifurcatedResource(
     isLovellFederalResource(resource as LovellBifurcatedFormattedResource)
   )
 }
+
+export function getLovellVariantOfMenu(menu, variant: LovellVariant) {
```

Comment:
If it's possible to type `menu` I think that would be helpful for understanding at a glance. It looks like this is recursing through `DrupalMenuLinkContent[]` or a `MenuItem`, is that right?

Reply:
Good call, fixed